src/main/java/uz/bookstore/bookstore/controller/AddressController.java

```diff
@@ -0,0 +1,30 @@
+package uz.bookstore.bookstore.controller;
```

Comment:
We do not need abstraction in controllers. We need only controller not interface and implementation.